changelogs/fragments/20230915_migrate_iam_role_and_iam_role_info.yml

```diff
@@ -0,0 +1,7 @@
+breaking_changes:
```

Comment:
Is it a breaking or major change?

Reply:
Given that we have the redirects in place I'd argue it's either "Major" (just for the sake of highlighting it) or potentially even "Minor".